data/hails.txt

```diff
@@ -974,6 +997,7 @@ phrase "friendly civilian"
 		" "
 	word
 		"trade routes, but I'm not"
+		"trade routes, but there is no way I'm"
```

Comment:
Could we change "there is" to "there's"? It's correct, but it sounds a bit awkward to me.